### PR TITLE
Add extra copy-project-relative-path replacing path separators for windows

### PR DESCRIPTION
**Pull request by s-shin**
https://github.com/s-shin/atom-copy-path/pull/7

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -21,6 +21,49 @@ module.exports = new class {
...
+    if (process.platform === "win32") {
+      this.activateForWin();
+    }
+  }
+
+  activateForWin() {
+    this.subscriptions.add(atom.commands.add("atom-workspace", {
+      "copy-path:copy-project-relative-path-web": (e) => this.copyProjectRelativePathForWeb(e)
+    }));
+    atom.packages.onDidActivatePackage((package) => {
```
Comment (s-shin) [inline lib/index.js:33]: A bit hacky way but there would be no way to separate menu cson files between platforms.
cf. Package#loadMenu: https://github.com/atom/atom/blob/v1.5.3/src/package.coffee#L247-L252

**Comment by fweber-de** [comment]
hold on, it does not work yet, will check again

---

```
looks good to me and it works ;)
i will close my PR then

can you notify me when the package is distributed?
```

**Comment by fweber-de** [comment]
ok, sorry for the disruption, it was just a weird error message but i couldnt reproduce it

@s-shin ist works, looks good to merge :)

**Comment by s-shin** [comment]
Thanks, will publish soon!

**Pull request merged**